app/controllers/users.js

```diff
@@ -41,3 +41,23 @@ exports.getUsers = async (req, res, next) => {
     next(error);
   }
 };
+exports.adminLog = async (req, res, next) => {
+  try {
+    const userObject = req.body;
+    const userFound = await userService.findUser(userObject);
+    if (userFound === null) {
```

Comment:
```suggestion
    if (!userFound) {
```

Read [this guide](https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#41--ifs)